### PR TITLE
AlliCo Hallmark Goods

### PR DESCRIPTION
**Pull request by Schnayy**
Adds the gift vendor, AlliCo Hallmark Goods. Is a delightfully gaudy vending machine placed in the locker rooms. The coffee machine was removed and the sweatmax moved to take it's place so the gift vendor is next to the laptop and cigarette vendors. Prices were based off of the SweatMax vendor.

Before:
image

After: 
image

Vendor will normally dispense:
- Heart box chocolates
- Bouquets
- Cheaper fake bouquets
- Each of the 4 cards previously added.
- All the therapy dolls.

Will dispense the following with a coin:
- All small plushies
- Newly added trinket box.

Will dispense the following when hacked:
- Gilthari Luxury Champagne
- Plans to add a handbuzzer in the future but not right now.

Items & Reagents added:

- Champagne. Is not a mixable reagent, listed as a child of wine.
- Gilthari Luxury Champagne. Contains said reagent.
- Trinket box. Think an engagement ring box, except it can also hold coins and medals. May be toggled open/closed, when opened it's description will say what's inside. Can only hold one item. Thanks to Mechoid for helping me code this mess. 
image
- Also goes in and fixes a small bug with the heart box saying it contains donuts when it does, in fact, not contain donuts.

Side note, mapping things in is scary but I did update my git late last night after those medbay map changes were done & checked so it SHOULD be okay?

### CODE REVIEW

**Comment by zaredman** [comment]
Looking at the mapping there - Did you use map merger? Seeing over 10k additions/removals which makes me think it was not used.

**Comment by Schnayy** [comment]
I... I think I fixed it..........

**Comment by Anewbe** [comment]
I'm not really sold on the name? I've definitely been to a Hallmark before.

**Comment by Schnayy** [comment]
It was intended as the actual word, not company name, sort of like 'luxury' in 'Gilthari Luxury Champagne' but I can change it if you'd like.

**Comment by Arokha** [comment]
> Did you use map merger?

@zaredman Mapmerge 2... switch to tgm format, you know you want to. 👀

**Comment by Schnayy** [comment]
Will make corrections/fixes tonight/tomorrow morning.

**Comment by Schnayy** [comment]
Trinket box has been committed out & items.dmi reverted as I was getting a bizarre conflict. Will be immediately readded in a separate PR.

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -1084,4 +1084,47 @@
...
+					/obj/item/weapon/paper/card/heart = 1,
+					/obj/item/weapon/paper/card/cat = 1,
+					/obj/item/weapon/paper/card/flower = 1,
+					/obj/item/clothing/accessory/bracelet/friendship = 5,
+					/obj/item/toy/plushie/therapy/red = 20,
+					/obj/item/toy/plushie/therapy/purple = 20,
+					/obj/item/toy/plushie/therapy/blue = 20,
+					/obj/item/toy/plushie/therapy/yellow = 20,
+					/obj/item/toy/plushie/therapy/orange = 20,
+					/obj/item/toy/plushie/therapy/green = 20)
```
Comment (Atermonera) [inline code/game/machinery/vending.dm]: Didn't close this list

Comment (Schnayy) [reply code/game/machinery/vending.dm]: ... i did a silly

**Comment by Atermonera** [comment]
Conflict arose from somewhere over the plurality of "chocolate" in the heartbox's icon_type. It has been squashed

**Pull request merged**